web_src/css/shared/flex-list.css

```diff
@@ -68,6 +80,15 @@
   overflow-wrap: anywhere;
 }
 
+/* display:inline wrapper that makes labels overflow like text */
+.flex-item-title-inline {
+  display: inline;
+}
+.flex-item-title-inline > * {
+  display: inline !important;
```

Comment:
It doesn't look good to me.

1. It introduces unclear style overriding game again.
2. There could be "SVG/emoji" in the labels, so the "flex align item center" was done intentionally, but now I do not know whether "inline+vertical-align" works.